python/tritonserver/_c/tritonserver_pybind.cc

```diff
@@ -1434,7 +1434,18 @@ class PyServer : public PyWrapper<struct TRITONSERVER_Server> {
     owned_ = true;
   }
 
-  void Stop() const { ThrowIfError(TRITONSERVER_ServerStop(triton_object_)); }
+  void Stop() const
```

Comment:
@rmccorm4 would it make sense to force a garbage collector run here before releasing?

Reply:
just a thought as it might be a good spot to do it semi-deterministically